### ISSUE TITLE
DatePicker: TalkBack announces superfluous text when navigating dates

### ISSUE THREAD

**Issue by ankushgoyal27** [angular/components#20777]
#### Reproduction

Steps to reproduce:
1. Open the Calendar in Android Chrome browser from Angular material site
https://material.angular.io/components/datepicker/overview
2. Enable talkback 
 
#### Expected Behavior

What behavior were you expecting to see?
It should announce the Date Month and Year.
i.e 13th October 2020 column 3

#### Actual Behavior

What behavior did you actually see? 
It is announcing the Date Month and Year and then append disabled text. 
i.e 13th October 2020 Disabled column 3

#### Environment

- Angular: 10.1.0
- CDK/Material: 10.1.3
- Browser(s): Chrome
- Operating System (e.g. Windows, macOS, Ubuntu):  Android

**Comment by ankushgoyal27** [comment]
Hi When can i expect the solution for this. Thanks

**Comment by zarend** [comment]
I tried this on talkback and the calendar cells are announced as toggle buttons without the word "Disabled". Looks like this was fixed by #24171
